floodviz/hydrograph_utils.py

```diff
@@ -19,13 +20,17 @@ def parse_hydrodata(jdata):
 
     """
     all_series_data = []
-    
+    # gages can collect and send data every 15 minutes
```

Comment:
This comment is not true. Maybe instead indicate that this code will fill in data gaps of more the 30 minutes